.golangci.yml

```diff
@@ -7,6 +7,18 @@ run:
   # timeout for analysis, e.g. 30s, 5m, default is 1m
   timeout: 10m
 linters-settings:
+  depguard:
+    rules:
+      # Name of a rule.
+      main:
+        # Used to determine the package matching priority.
+        # There are three different modes: `original`, `strict`, and `lax`.
+        # Default: "original"
+        list-mode: lax
+        # List of allowed packages.
+        allow:
+          - $gostd
+          - github.com/ONSDigital
```

Comment:
Our general convention on this is just to disable `depguard` given we don't leverage it's functionality anyway.

Reply:
depguard has now been removed but do you know if this decision was documented anywhere?

Reply:
I'm not sure it was - should be added to the standard linting setup I guess.